src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+
+public class Main {
+    private static final int amountOfCreatedFigures = 5;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[amountOfCreatedFigures];
+        for (int i = 0; i < amountOfCreatedFigures; i++) {
+            if (i == 1) {
```

Comment:
The first half of figures in this array should be generated with random parameters.

Reply:
not fixed